### PR TITLE
Lowering volume drastically of the pike pole

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
Balance "Lowering the volume of the pike poke to more sensible values"

#### Purpose of change
I didn't want to squeze it in the other pull request, so it got seperated. The volume of over 3 liters for such glorified long stick with a hook on top is more then overkill. Let's do some math. If the pole itself has a diameter of around 3 cm which is thick, to be frank, then we half it for the radius -> 1,5 cm, square it -> 2,25, mulitplicate with pi -> ~7,07 cm² multiplicate with 160 cm lengh -> 1130 cm³ which is 1,13 liters. We add the hook which is around 100 ml of volume and we get the number I got. Far away from the old 3 liters! Else the pole would be 4 meters long!

### CODE REVIEW

**Comment by actual-nh** [comment]
The extra "round up" of 20 mL can be for a cloth to keep the hook from snagging on anything... neatness in-game as well as in-file!

File: data/json/items/tool/firefighting.json [hunk 1]
```diff
@@ -192,7 +192,7 @@
     "techniques": [ "WBLOCK_1", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "REACH_ATTACK", "NONCONDUCTIVE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
     "weight": "3100 g",
-    "volume": "3250 ml",
+    "volume": "1250 ml",
```
Comment (anothersimulacrum) [inline data/json/items/tool/firefighting.json:195]:
```suggestion
    "//": " If the pole itself has a diameter of around 3 cm which is thick, to be frank, then we half it for the radius -> 1,5 cm, square it -> 2,25, mulitplicate with pi -> ~7,07 cm² multiplicate with 160 cm lengh -> 1130 cm³ which is 1,13 liters. We add the hook which is around 100 ml of volume."
    "volume": "1250 ml",
```

Comment (Lamandus) [reply data/json/items/tool/firefighting.json:196]: thanks that my comment is "json comment" worthy.

File: data/json/items/tool/firefighting.json [hunk 2]
```diff
@@ -192,7 +192,8 @@
     "techniques": [ "WBLOCK_1", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "REACH_ATTACK", "NONCONDUCTIVE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
     "weight": "3100 g",
-    "volume": "3250 ml",
+    "//": " If the pole itself has a diameter of around 3 cm which is thick, to be frank, then we half it for the radius -> 1,5 cm, square it -> 2,25, mulitplicate with pi -> ~7,07 cm² multiplicate with 160 cm length -> 1130 cm³ which is 1,13 liters. We add the hook which is around 100 ml of volume."
```
Comment (anothersimulacrum) [inline data/json/items/tool/firefighting.json:195]: Whoops, my bad.
```suggestion
    "//": " If the pole itself has a diameter of around 3 cm which is thick, to be frank, then we half it for the radius -> 1,5 cm, square it -> 2,25, mulitplicate with pi -> ~7,07 cm² multiplicate with 160 cm length -> 1130 cm³ which is 1,13 liters. We add the hook which is around 100 ml of volume.",
```

**Pull request merged**